content/md/productivity/manage-your-rules.md

```diff
@@ -0,0 +1,225 @@
+
+The first time a rule is executed, it can match many products. The following time, you might have zero `Impacted products` or only a few.
+
+1.  To use this action, select one or several rules by ticking them on the left side of the screen
+1.  Click on `Calculate the impacted products`
+1.  Then confirm the action in the pop in
+1.  A flash message confirming the action will be displayed and you will be notified once the calculation is done.
+
+![image](../img/Settings_RulesCalculate.png)
+
+The number of impacted products will appear on the right side of your screen in the `Impacted products` column.
+
+
+# Execute rules
+
+:::info
+Rules are regularly executed after a bulk action or after imports with rules.
+:::
+
+But you can also execute rules on your own. You can execute:
+*   one specific rule
+*   a selection of rules
+*   all your rules (be aware, this action could take a long time)
+
+:::info
+The rules support [product models](what-about-products-variants.html#what-is-a-product-model), so rules can be defined also on attributes at the product model level (or sub product model level).
+:::
+
+The rules execution summary is accessible via the `process tracker`.
+
+![image](../img/Rules_ProcessTracker.png)
+
+In this summary, you can easily access the details of the execution: how many rules were executed, how many products/product models were updated, warnings...
+
+![image](../img/Rules_Summary.png)
+
+:::tips
+In order to access the summary of nightly scheduled rules (by cron), you need to have the right to `View all jobs in process tracker` in the `Roles` permissions to be able to see them the process tracker :wink:.
+:::
+
+## Execute a single rule
+
+To execute a rule, click on the `Execute` button at the end of each line in the grid.  
+A confirmation message validates the action of executing a rule.
+
+![image](../img/Settings_RulesExecution1.png)
+
+
+## Execute a selection of rules
+
+You can also execute several rules at the same time:
+1.  Select one or several rules by ticking them on the left side of the screen
+1.  Click on `Execute`
+1.  Then confirm the action in the pop in
+1.  A flash message confirming the action will be displayed and you will be notified once the execution is done
+
+![image](../img/Settings_RulesExecutionNotif.png)
+
+# Delete rules
+
+::: warning
+A deleted rule can no longer be executed by the rules engine. The products keep the values calculated by the deleted rule.
+:::
+
+## Delete a single rule
+
+To delete a rule, click on the `Delete` button at the end of each line in the grid.  
+A confirmation message validates the action of deleting a rule.
+
+![image](../img/Settings_RulesDeleteHover.png)
+
+::: info  
+If you do not see the `Delete` button, it might be because you do not have the relevant rights. Please refer to [Manage your user roles](build-your-user-roles.html).
```

Comment:
```suggestion
If you do not see the `Delete` button, it might be because you do not have the right permissions. Please refer to [Manage your user roles](build-your-user-roles.html).
```